src/View/Helper/ListsHelper.php

```diff
@@ -337,9 +337,23 @@ public function displayVisibilityOption($listId, $value)
             ?>
             <input type="radio"  name="visibility" data-list-id='<?= $listId ?>'  value="{{visibility}}" checked hidden ng-init="visibility = '<?= $value ?>';"/>
             <md-radio-group ng-controller='optionsCtrl' ng-model='visibility' ng-change='visibilityChanged()'>
-                <md-radio-button value='public' class='md-primary'>
+                <md-radio-button value='public' class='md-primary' title=
+                                 '<?=h(__("The list will be shown on the sentences page"))?>'>
                     <?php /* @translators: visibility option of a list */ ?>
-                    <?=  __('Public') ?>
+                    <?=  __('Public'); 
+                    /* @translators: visibility option of a list *                  
+                    /*should be removed after 2020*/
+                    echo '<span style="color:red">' . __(' new');                    
+                    /* @translators: notice of new visibility feature of a list */
+                    
+                    ?>
+                    </span>
```

Comment:
This code works, but I think it would be easier to read if you leave out the markup out of the PHP `<? ?>` tags. For example:
```html
<?= __('Public') ?>
<span style="color:red"><?= __('new') ?></span>
```

Reply:
So you included a space before the string "new". Although it looks okay, we’d like to avoid that. Instead, try to use CSS positioning to move the string where you want it to be. This is because in HTML, the contents (characters, including the space character) should be clearly [separated from the presentation](https://en.wikipedia.org/wiki/Separation_of_content_and_presentation).

Reply:
About the `@translators` note, I think it’s not exactly doing what you think it is. Only the note *above* the `__()` call is taken into account, not underneath. You can see by yourself how the note is interpreted by running the strings extraction script `./tools/generate_pot.sh` and then `git diff` to check how the file `src/Locale/default.pot` was modified. With your current code, the string is extracted like this (this is an excerpt from `src/Locale/default.pot` after running `./tools/generate_pot.sh`):
```pot
#. visibility option of a list *
#. /*should be removed after 2020
#: github.com/Tatoeba/tatoeba2/tree/dev/src/View/Helper/ListsHelper.php#L346
msgid " new"
msgstr ""
```
So we end up with this note:
```
visibility option of a list *
/*should be removed after 2020
```